spec/system/planning_applications/checking_consistency_spec.rb

```diff
@@ -295,7 +288,8 @@ def form_group_with_legend(legend)
     find("legend", text: legend).find(:xpath, "..")
   end
 
-  def task_list_item_with_name(name)
-    find("span", text: name).find(:xpath, "..")
+  def task_list_item
+    text = "Description, documents and proposal details"
+    find("span", text: text).find(:xpath, "..")
```

Comment:
Not sure if this is worth extracting out into the system spec helpers ?

And again making it dynamic if perhaps we need to use it for another spec. But not fussed if you think it's not necessary